pkgs/top-level/all-packages.nix

```diff
@@ -6320,8 +6320,6 @@ with pkgs;
 
   ### DEVELOPMENT / TOOLS
 
-  activator = callPackage ../development/tools/activator { };
```

Comment:
done. It gives me now the following output when trying to build:

```
→ nix-build -A activator
error: Typesafe Activator was removed in 2017-05-08 as the actual package reaches end of life.

See https://github.com/NixOS/nixpkgs/pull/25616
and http://www.lightbend.com/community/core-tools/activator-and-sbt
for more information.
```

I've squashed it into the previous commit as it directly belongs to this change.